app/assets/src/components/common/DetailsSidebar/GeneDetailsMode/GeneDetailsMode.jsx

```diff
@@ -0,0 +1,430 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { getCARDIndex, getAroEntry } from "~/api/amr";
+
+import cs from "./gene_details_mode.scss";
+
+const SOURCE_CARD = "card";
+const SOURCE_PUBMED = "pubmed";
+const SOURCE_GOOGLE_SCHOLAR = "googlescholar";
+const SOURCE_NCBI_REF_GENE = "ncbirefgene";
```

Comment:
nit: up to you but the value would be more readable as ncbiRefGene?

Reply:
Made even more readable!